packages/ui/src/hooks/useCoinConversion.ts

```diff
@@ -20,6 +20,8 @@ export default function (
 
   const baseUrl = createBaseUrl(providerOptionsContext?.coinGecko)
 
+  id = id ? id : providerOptionsContext.coinGecko?.coinIds?.[symbols] || ''
```

Comment:
I guess it's called symbols because you can pass multiple? So we would probably need to split on any commas and then get each id and join everything back up to pass to the coingecko api.